src/cfg/cfg-traversal.h

```diff
@@ -205,47 +211,79 @@ struct CFGWalker : public ControlFlowWalker<SubType, VisitorType> {
     self->startUnreachableBlock();
   }
 
-  static void doEndCall(SubType* self, Expression** currp) {
+  static void doEndThrowingInst(SubType* self, Expression** currp) {
     // Every call can possibly throw, but we don't end the current basic block
```

Comment:
`call` is no longer accurate throughout this comment

Reply:
Changed to
```
Even if the instruction can possibly throw, we don't end the current basic block unless ...
```